Mathlib/Probability/Integration.lean

```diff
@@ -164,6 +164,7 @@ theorem IndepFun.integrable_left_of_integrable_mul {β : Type*} [MeasurableSpace
     apply IndepFun.comp hXY M M
   have A : (∫⁻ ω, ‖X ω * Y ω‖₊ ∂μ) < ∞ := h'XY.2
   simp only [nnnorm_mul, ENNReal.coe_mul] at A
+  simp_rw [enorm_eq_nnnorm] at H
```

Comment:
There might be a more elegant solution: pointing this out just in case.

Reply:
Changed the proof slightly to not unfold the definition of `HasFiniteIntegral`.